IPython/core/inputtransformer2.py

```diff
@@ -481,6 +481,17 @@ def make_tokens_by_line(lines):
     if not tokens_by_line[-1]:
         tokens_by_line.pop()
 
+    # Convert if using cpython tokenize
```

Comment:
```suggestion
    # Convert if using cpython tokenize
    # upstream bug was fixed in Python 3.7.1, so once we drop 3.7 this can likely be removed. 
```